See this exception on Errbit: <%= app_err_url err.app, err %>
<% if notice = err.notices.first %>
    <% if notice.request['url'].present? %>URL: <%= notice.request['url'] %><% end %>
    Where: <%= notice.err.where %>
    Occurred: <%= notice.created_at.to_s :micro %>
    Similar: <%= (notice.err.notices.count - 1).to_s %>

    Params:
    <%= pretty_hash notice.params %>

    Session:
    <%= pretty_hash notice.session %>

    Backtrace:
    <%= notice.backtrace.map { |line| "#{line['number']}: #{line['file'].sub(/^\[PROJECT_ROOT\]/, '')} -> *#{line['method']}*" }.join "\n" %>

    Environment:
    <% notice.env_vars.each do |key, val| %>
        <%= "#{key}: #{val}" %>
    <% end %>
<% end %>
